### PR TITLE
products/alinux2: Add CIS Alibaba Cloud Linux (Aliyun Linux) 2 profiles

### PR DESCRIPTION
**Pull request by hustliyilin**
#### Description:

- CIS Aliyun Linux 2 Benchmark v1.0.0
(https://workbench.cisecurity.org/benchmarks/2228) was published
on Aug 16th 2019. Aliyun Linux 2 is compatible with CentOS 7 and it's
further renamed as Alibaba Cloud Linux 2.

#### Rationale:

- I add the CIS Alibaba Cloud Linux (Aliyun Linux) 2 controls in the OpenSCAP
according to CIS Aliyun Linux 2 Benchmark v1.0.0
(https://workbench.cisecurity.org/benchmarks/2228)

- Fixes: #8895

### CODE REVIEW

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Hello, thank you for this new profile. Well done.
There is one thing I would like to ask you to do:
When you use a rule within the new profile, it needs to get also the correct CIS reference.
For example the rule kernel_module_squashfs_disabled is assigned to CIS control 1.1.1. So you need to add 
cis@alinux2: 1.1.1
into the "references" section. Please note that you have to keep references in alphabetical order.
I believe that after this change the PR will be reade to merge.

File: controls/cis_alinux2.yml
Comment (vojtapolasek) [inline controls/cis_alinux2.yml:4]:
```suggestion
version: '1.0.0'
```

File: products/alinux2/profiles/cis.profile
Comment (vojtapolasek) [inline products/alinux2/profiles/cis.profile:3]: We would like to have note who is the maintainer of a profile. Therefore, we use the "SMEs" section in profile metadata.
You can see an example here:
https://github.com/ComplianceAsCode/content/blob/master/products/rhel8/profiles/cis.profile
Could you add this section with at least one github handle?

**Review by vojtapolasek — COMMENTED** [review]
Looks nice. Just some nitpicks and it will be ready to be merged.

**Comment by hustliyilin** [comment]
> Looks nice. Just some nitpicks and it will be ready to be merged.

Hi @vojtapolasek ,

Thanks for your review. I already updated the codes and force-pushed this pr. Could you review it again ? Thanks a lot.

**Pull request merged**
